src/dlgTriggerEditor.cpp

```diff
@@ -2843,15 +2843,9 @@ void dlgTriggerEditor::activeToggle_trigger()
         }
     }
 
-    if (pT->state()) {
-        if (pT->shouldBeActive()) {
-            showInfo(tr(R"(Trying to activate a trigger group, filter or trigger or the part of a module "<tt>%1</tt>" that contains them <em>succeeded</em>.)").arg(pT->getName().toHtmlEscaped()));
-        } else {
-            showInfo(tr(R"(Trying to deactivate a trigger group, filter or trigger or the part of a module "<tt>%1</tt>" that contains them <em>succeeded</em>.)").arg(pT->getName().toHtmlEscaped()));
-        }
-    } else {
+    if (!pT->state()) {
         pT->setIsActive(false);
-        showError(tr(R"(<b>Unable to activate a filter or trigger or the part of a module "<tt>%1</tt>" that contains them; reason: %2.</b></p>
+        showError(tr(R"(<p>Unable to activate "<tt>%1</tt>"; %2</p>
```

Comment:
:thinking: is it acceptable to lose the longer text, it is possible (but not ideal) IIRC for the text that is inserted as %2 to not be in the GUI language - so having something like:
"**This is broken; the reason things don't work is:** *you have not put any X into box Y*".

is a bit better UI in my humble opinion than the equivalent to:
"This is broken; Il y a un problème entre la chaise et le clavier."

Reply:
The shorter text does look better in the context of the UI, and it has been appreciated by players: https://discord.com/channels/283581582550237184/283582439002210305/1277272537159434281

Reply:
This text is a sentence - does it not need to end in a '.' - at least for the Engineering (American) English? 

Good catch on the missing opening `<p>` but since it contains HTML tags anyway (the `<tt>...</tt>` to format the name of the item) the paragraph tags are not essential as the Qt Rich-text detection will fire off of those other tags...

Reply:
User interface texts don't need to be proper sentences ending with a period, e.g: 
![image](https://github.com/user-attachments/assets/2911c491-ecec-4b91-9aca-35dc4f701362)

Reply:
Maybe not in things like that - but consider a related error message - where there is a completely separate sentence explaining further action the user should take - are you going to take out the full stop of the first sentence and run it into the second and lose the terminal punctuation for that as well - or are we going to end up with a mixture:
```cpp
        showError(tr(R"(<p><b>Unable to activate a button/menu/toolbar or the part of a module "<tt>%1</tt>" that contains them; reason: %2.</b></p>
                     <p><i>You will need to reactivate this after the problem has been corrected.</i></p>)").arg(pT->getName().toHtmlEscaped(), pT->getError()));
```

Reply:
This is not really how modern UX design tends to structure texts, so let's go with the original flow.

Reply:
In that case retain the original colon as an "introducer" to the error message that follows rather than a semi-colon which just suggests a larger break in flow than a comma: (<- see that one there - it would be doing exactly this if I hadn't diverted your attention with this comment!)
```suggestion
        showError(tr(R"(<p>Unable to activate "<tt>%1</tt>": %2</p>
```